search/views.py

```diff
@@ -26,25 +26,38 @@ class ElasticProxyView(APIView):
         SessionAuthentication,
         TokenAuthentication,
     )
-    permission_classes = (IsAuthenticated, UserCanSearchPermission, )
+    permission_classes = (IsAuthenticated,)
 
-    def _execute_search_from_request(self, user, request_data):
+    def _is_advance_search_capable(self):
+        """
+        Determines if a user can perform advanced search
+        """
+        return UserCanAdvanceSearchPermission().has_permission(self.request, self)
```

Comment:
Discussed w/ Peter, we decided that if you're staff in any programs we should limit search results to the set of programs you're staff in. That effectively means that staff won't be able to use this leaner search for a program they're not staff in, but we're ok with that limitation if it preserves the current privacy model (which it does).